tests/conftest.py

```diff
@@ -1,5 +1,17 @@
 # SPDX-License-Identifier: LGPL-2.1-or-later
 
+import pytest
+
+
 pytest_plugins = [
     "bluechi_test.fixtures"
 ]
+
+
+# Set minimum timeout for all tests to 30 seconds.
+# If some test needs bigger timeout, please override it for the specific test
```

Comment:
did we want to bump the ones changed here to 45 sec? 10/20/40 wasn't enough?
Or were we just trying to be consistent?

Reply:
According to @Yarboa 45 seconds timeout improved a lot test result consistency: https://github.com/containers/bluechi/pull/516